docs/source/python/api/compute.rst

```diff
@@ -388,18 +467,9 @@ Structural Transforms
 .. autosummary::
    :toctree: ../generated/
 
-   binary_length
-   case_when
-   choose
-   coalesce
-   fill_null
-   if_else
-   is_finite
-   is_inf
-   is_nan
-   is_null
-   is_valid
    list_element
    list_flatten
    list_parent_indices
    list_value_length
+   make_struct
+   replace_with_mask
```

Comment:
(not that important but small note in case you would update this file in the future again: I don't think `replace_with_mask` is a "structural transform", since it only replaces values inside the array and doesn't change the array type or structure in any way)

Reply:
I agree. The reason for these changes is because in C++ docs categorize kernels based on their kind (aggregate, scalar, vector) but Python docs do not. In C++, `make_struct` is listed as a scalar structural transform and `replace_with_mask` as a special case of vector structural transforms](https://github.com/apache/arrow/pull/11237/files#diff-ce5b94577014735990903d3d03bd4ea4b8c8e6d32f5227592e60b7dd6a912d59R1544). In Python docs, I place them under the unified structural transform category (`replace_with_mask` was missing).